tests/test_debugging.py

```diff
@@ -503,6 +504,144 @@ def test_has_active_device(self):
         PLDB.reset_active_dev()
         assert not PLDB.has_active_dev()
 
+    tapes = (
+        qml.tape.QuantumScript(
+            ops=[qml.Hadamard(0), qml.CNOT([0, 1])],
+            measurements=[qml.state()],
+        ),
+        qml.tape.QuantumScript(
+            ops=[qml.Hadamard(0), qml.X(1)],
+            measurements=[qml.expval(qml.Z(1))],
+        ),
+        qml.tape.QuantumScript(
+            ops=[qml.Hadamard(0), qml.CNOT([0, 1])],
+            measurements=[qml.probs()],
+        ),
+        qml.tape.QuantumScript(
+            ops=[qml.Hadamard(0), qml.CNOT([0, 1])],
+            measurements=[qml.probs(wires=[0])],
+        ),
+        qml.tape.QuantumScript(
+            ops=[qml.Hadamard(0)],
+            measurements=[qml.state()],
+        ),  # Test that state expands to number of device wires
```

Comment:
This comment belongs to here?

Reply:
Yes, I am testing the case where we specified more wires on the device than we used in the circuit. Asking for `qml.debugging.state()` should show you all wires specified on the device, not just the ones we used. 

```
dev = qml.device("default.qubit", wires=2)
@qml.qnode(dev)
def circ():
    qml.Hadamard(0)
    return qml.state()  # or qml.breakpoint() followed by qml.debugging.state()

circ()  # The state here will have 2 wires, even though only one is used in the circuit 
           # because the device has 2 wires.
```